doc/src/content/xdocs/gettingstartedjava.xml

```diff
@@ -440,7 +446,13 @@ System.out.println(user);
           converts in-memory serialized items into <code>GenericRecords</code>.
           We pass the <code>DatumReader</code> and the previously created
           <code>File</code> to a <code>DataFileReader</code>, analogous to the
-          <code>DataFileWriter</code>, which reads the data file on disk.
+          <code>DataFileWriter</code>, which reads both the schema used by the
+          writer as well as the data from the file on disk. The data will be
+          read using the schema included in the file, and converted to the
```

Comment:
This is very similar to above, so whatever you change there should also be applied here.